doc/usage.rst

```diff
@@ -89,6 +89,30 @@ Or in Windows, use the little-known BAT file line continuation::
         myscript.spec
 
 
+Running PyInstaller with Python optimizations
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
The underlining must be at least as long (in number of characters) as the heading.